extensions/2.0/Vendor/EXT_mesh_features/schema/gltf.EXT_feature_metadata.schema.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Make sure the name of this file gets updated to `gltf.EXT_mesh_features.schema.json` -- there are a couple other files like this too.